arise

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+# https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
I think the reason you modified this and moved it up here is to allow a config option to make it easier to overwrite To make this cleaner, I changed it to an optional item; see additional suggestion below to use the snippet by default but allow overwrite. 

Since PR base branch changes aren't disabled and GitHub reviews won't let me comment on the entire file, can I ask that you move this option below the version tag stub but above the 'begin main function' stub?

```suggestion
# If you want to manually specify the path to the arise binary, you can uncomment and modify the variable below.
# This should not be necessary in most cases and this should generally be left as default.
# SCRIPT_DIR="/change/this/to/path/to/arise"
```